swift/QuartzStyles/QuartzStyles.swift

```diff
@@ -69,4 +69,5 @@ public enum QuartzStyles {
     public static let sizeGutterMobile = CGFloat(20.00)
     public static let sizeGutterTablet = CGFloat(40.00)
     public static let sizeGutterTabletWork = CGFloat(25.00)
+    public static let timeSlowly = 300
```

Comment:
This is kinda weird as an integer, I might have to write a custom transformer to convert this to a native time type. 
But as long as this compiles 👍  for now

Reply:
Hmm, not sure why it's not being smarter about that. It uses the [time/seconds transformer](https://amzn.github.io/style-dictionary/#/transforms?id=timeseconds), which converts to `0.3s` for SCSS